generate/templates/couchbase-edge-server/Dockerfile.template

```diff
@@ -0,0 +1,26 @@
+FROM {{ .DOCKER_BASE_IMAGE }}
+
+LABEL maintainer="docker@couchbase.com"
+
+ENV PATH $PATH:/opt/couchbase-edge-server/bin
+
+# Install dependencies:
+RUN set -x \
+    && apt update \
+    && apt install -y \
+           curl \
+           lsb-release \
+           systemctl \
+           wget \
+    && apt clean
+
+# Install Couchbase-Edge-Server
+ARG EDGE_SERVER_RELEASE_URL="{{ .CB_RELEASE_URL }}"
+ARG EDGE_SERVER_PACKAGE_FILENAME="{{ .CB_PACKAGE_NAME }}"
+RUN set -x \
+    && EDGE_SERVER_PACKAGE_FILENAME=$(echo ${EDGE_SERVER_PACKAGE_FILENAME} | sed -e "s/@@ARCH@@/$(dpkg --print-architecture)/") \
+    && wget ${EDGE_SERVER_RELEASE_URL}/${EDGE_SERVER_PACKAGE_FILENAME} \
+    && apt install -y ./${EDGE_SERVER_PACKAGE_FILENAME} \
+    && rm ${EDGE_SERVER_PACKAGE_FILENAME} \
+    && apt autoremove \
+    && apt clean
```

Comment:
This is going to need more thought than this.  It's going to look pretty similar to Sync Gateway's docker config file in that it needs to have a bootstrap, etc, to pass arguments and what not

Reply:
Need at least to have an `COPY scripts/entrypoint.sh /entrypoint.sh` and `ENTRYPOINT ["/entrypoint.sh"]` directives in the Dockerfile.

Reply:
@borrrden, a few questions.

- we need a readme file similar to https://github.com/couchbase/docker/blob/master/generate/resources/sync-gateway/README.md.  Who will be able to produce this?
- Is there any port the dockerfile need to expose?